contracts/requests/WitnetRequest.sol

```diff
@@ -28,6 +30,7 @@ abstract contract WitnetRequestTemplate
     function getRadonTally() virtual external view returns (WitnetV2.RadonReducer memory);
     
     function buildRequest(string[][] calldata args) virtual external returns (WitnetRequest);
+    function verifyRadonRequest(string[][] calldata args) virtual external returns (bytes32);
```

Comment:
The "query" / "request" concepts are not actually new of this PR:

- "Request" actually refers to "DataRequestOutput" as defined in Protobuf, which is internally composed of a RADRequest (uniquely identified in WitnetBytecodes and the Witnet block explorer by the `radHash`) and the witnessing params (uniquely identified in WitnetBytecodes as `slaHash`). 

- "Query" doesn't actually refer to some pair RAD + SLA, but to some DataRequestOutput (RAD + SLA) posted to the WRB from within certain EVM transaction hash. Two postings of same RAD + SLA within the same EVM block would actually generate different "WRB query ids". The "query id" is just a way to uniquely identify a RAD+SLA postings to the WRB, and it's intended to be used by (a) the RAD+SLA poster (the "requester"), (b) the bridge in charge of broadcasting the DataRequestTransaction to the Witnet network, and (c) the bridge in charge of reporting back the result to previous transaction back to the WRB. Atm, (b) and (c) are in practice the same identity. In a more decentralized and/or permissionless model for bridges, (b) and (c) could eventually happen to be different actos.

Reply:
So if we wanted to be strict with the naming, may be renaming `postRequest` to `postDataRequestOutput` would be more appropiate than renaming `_queryId` to `_requestId`,  but then again it would not only be more verbose, but also misleading for the regular developer who should not need to be aware of the intricacies and low-level concepts of the Witnet protobuf-based protocol.

Reply:
Concerning the `verifyRadonRequest` method within the WitnetRequestTemplate and the WitnetBytecodes contracts, it should definitely remain named as such, for it's a RAD+SLA what's it's intended to be validated (and registered) within the (underlying) WitnetBytecodes registry, but just a `RADRequest` as defined in protobuf.

Reply:
**witnet.proto**:
```
message DataRequestOutput {
    message RADRequest {
        enum RADType {
            Unknown = 0;
            HttpGet = 1;
            Rng = 2;
            HttpPost = 3;
        }
        message RADFilter {
            uint32 op = 1;
            bytes args = 2;
        }
        message RADRetrieve {
            RADType kind = 1;
            string url = 2;
            bytes script = 3;
            // Body of HTTP-POST request
            bytes body = 4;
            // Extra headers for HTTP-GET and HTTP-POST requests
            repeated StringPair headers = 5;
        }
        message RADAggregate {
            repeated RADFilter filters = 1;
            uint32 reducer = 2;
        }
        message RADTally {
            repeated RADFilter filters = 1;
            uint32 reducer = 2;
        }

        uint64 time_lock = 1;
        repeated RADRetrieve retrieve = 2;
        RADAggregate aggregate = 3;
        RADTally tally = 4;
    }

    RADRequest data_request = 1;
    uint64 witness_reward = 2;
    uint32 witnesses = 3;
    uint64 commit_and_reveal_fee = 4;
    uint32 min_consensus_percentage = 5;
    uint64 collateral = 6;
}
```